modules/retail/nav-retail.adoc

```diff
@@ -10,9 +10,18 @@ ifndef::backend-pdf[]
 *** xref:retail-install-packages.adoc[Installing from Packages]
 *** xref:retail-install-setup.adoc[Setting up]
 *** xref:retail-deploy-terminals.adoc[Deploying Terminals]
+// Formulas
+** xref:retail-formulas.adoc[Formulas]
```

Comment:
This diff tells me that you deleted `retail-formulas.adoc`, but here is ist still referenced.

Reply:
otherwise fine.  assuming that there are no additional wording changes (I did not read all the text).

Reply:
Good spot! Fixed now :)

I haven't changed any text.